pkg/kubelet/rkt/rkt.go

```diff
@@ -1404,16 +1406,16 @@ func (r *Runtime) RunPod(pod *v1.Pod, pullSecrets []v1.Secret) error {
 
 	// RestartUnit has the same effect as StartUnit if the unit is not running, besides it can restart
 	// a unit if the unit file is changed and reloaded.
-	reschan := make(chan string)
-	_, err = r.systemd.RestartUnit(name, "replace", reschan)
+	restartChannel := make(chan string)
```

Comment:
the name before actually was for "result channel". It was used as the systemd result for both the `restart` and `stop`.

I suspect it was inspired by how the upstream code here names its variables (https://github.com/coreos/go-systemd/blob/34529cec056a9a08b7fe216c2deb674cc9c46419/dbus/methods_test.go#L223-L233)